opentelemetry.html.md.erb

```diff
@@ -64,32 +109,106 @@ otlp:
   compression: snappy
 ```
 
-### <a id='multiple-exporters'></a> Multiple exporters
+## <a id='metric-processor-config'></a> Metric Processor configuration
 
-The OTel Collector supports defining multiple exporters, each with its own configuration. For example, you can define
-two OTLP gRPC exporters that egress metrics to different destinations.
+Processors are used at various stages of a pipeline to pre-processes data before it is exported. By default, no processors are enabled. The order of the processors in a pipeline determines the order of the processing operations that the Collector applies to the signal. 
+
+Refer to the individual processor [documentation](https://github.com/loggregator/otel-collector-release/tree/main/src/otel-collector/vendor/go.opentelemetry.io/collector/processor) for more information.
+
+Currently, the <%= vars.app_runtime_abbr %> distribution of the OpenTelemetry Collector supports the following processors out of the box: `batch`, `memorylimiter`. You can configure processors using the `processors` section of the Collector configuration file.
 
 ```
-otlp:
-  endpoint: 203.0.113.10:4317
-otlp/another:
-  endpoint: 203.0.113.11:4317
-```
+processors:   
+  batch:  
+```     
+
+<p class="note important">
+<span class="note__title"><strong>Important</strong></span>
+You can specify additional experimental processors including <code>transform</code>  and <code>filter</code> after enabling the <code>experimental OpenTelemetry Collector features (beta)</code> checkbox.</p>
 
 ### <a id='authentication'></a> Authentication
 
 The <%= vars.app_runtime_abbr %> distribution of the OpenTelemetry Collector does not support configuration of
```

Comment:
I'm not sure if this is true or not. We don't include any, but I think maybe our current configuration munging would allow the extension configuration to pass through. 

Maybe just strike this whole section for now? or change it to "The distribution of the Otel Collector does not currently include any authenticator extensions. Please let us know if there are authenticator extensions you would find useful."